com.unity.multiplayer.mlapi/Tests/Runtime/MultiInstanceHelpers.cs

```diff
@@ -302,9 +310,12 @@ public static IEnumerator WaitForClientsConnected(NetworkManager[] clients, Coro
             }
             else
             {
-                foreach (var client in clients)
+                for (var i = 0; i < clients.Length; ++i)
```

Comment:
++i and i++ makes no difference in C#, I see a C++ programmer here :)
even in C++, compiler is usually clever enough by the context to optimize it though.

Reply:
I was actually going to say, yes, the compiler optimizes the difference anyway. But... even being aware of it (and having even proved it to others by comparing the generated assembly), I still do this out of habit.

The one place it does matter in C++ is iterators, where pre- and post-increment are different method calls so the compiler can't optimize the difference without potentially changing the result, which the compiler's forbidden from doing, so since it doesn't make any difference anywhere else but it does matter there, I've never been very motivated to break this habit. In one context it's beneficial, in others it's irrelevant, and it's never harmful.